### DISCUSSION TITLE
Status bar - hovered link URL

### DISCUSSION THREAD

**Discussion by vertigo220** [xiaoxiaoflood/firefox-scripts#174] [General Support] [answered]
Is it possible to make the URL text when hovering over a link show above the status bar instead of in/over it? Having it on top of the bar prevents the bar's use for actually holding addons.

Also, the default bar doesn't show the URL text with a dark theme. I had to change the `#status-text > #statuspanel-label` in status-bar.uc.js to `background-color: black !important;` Just wanted to mention that in case anyone else runs into the issue. Interestingly, this wasn't necessary for Waterfox, as it was automatically black.

**Comment by xiaoxiaoflood** [comment]
It was always possible to remove status text from statusbar using Customize Toolbar screen, as you can see in the second screenshot in main page. Your comment made me realize that this is not available anymore. I'll take a look.

**Comment by xiaoxiaoflood** [comment, marked as answer]
I just released an update.

To remove status text from statusbar, open `about:config` and set `userChromeJS.statusbar.appendStatusText`  to `false`.

Regarding the dark theme issue, I can't reproduce it here, might be something OS-specific.

**Reply by vertigo220** [reply]
Thanks, this worked perfectly!
